data/sql/updates/pending_db_world/rev_1518477962308987200.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1518477962308987200');
+
+#--Evilpriest #0003
+#--Added EoF from Heart-Shaped Box
+
+INSERT INTO `item_loot_template` (`entry`,`item`,`ChanceOrQuestChance`,`lootmode`,`groupid`,`mincountOrRef`,`maxcount`) VALUES
+('54537','49426','100','1','0','2','2')
```

Comment:
you need to `DELETE` before insert, the SQL queries should be re-executable when possible.